functions/New-DbaDirectory.ps1

```diff
@@ -76,9 +76,9 @@ function New-DbaDirectory {
 
         $Path = $Path.Replace("'", "''")
 
-        $exists = Test-DbaPath -SqlInstance $SqlInstance -SqlCredential $SqlCredential -Path $Path
+        $exists = Test-DbaPath -SqlInstance $Instance -SqlCredential $SqlCredential -Path $Path
 
-        if ($exists) {
+        if ($exists.FileExists) {
```

Comment:
The purpose of the command is to create a directory and not a file. `$FileExists` will always be false if they provide a path with a file and not an explicit container.

It depends on what bug you are trying to fix as to what this needs to be changed to but overall the original check is fine because we do not output an object if the path provided does not exists.